FSMContextProxy has invalid update() function

## Context

With aiogram branch dev-2.x function `FSMContext.update` with kwvalues doesn't work

## Expected Behavior
```python
    async def test_update_data(self):
        context = FSMContext(MemoryStorage(), chat=1, user=1)
        async with context.proxy() as data:
            data.update(key1="value1", key2="value2")
```
Should run without exception

## Current Behavior
```python
FAILED [ 66%]
tests\test_dispatcher\test_fsm_context.py:6 (TestFSMContext.test_update_data)
self = <test_fsm_context.TestFSMContext object at 0x000001AABEC06820>

    @pytest.mark.asyncio
    async def test_update_data(self):
        context = FSMContext(MemoryStorage(), chat=1, user=1)
        async with context.proxy() as data:
>           data.update(key1="value1", key2="value2")

test_dispatcher\test_fsm_context.py:11: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

self = <aiogram.dispatcher.storage.FSMContextProxy object at 0x000001AABEC06A30>
data = None, kwargs = {'key1': 'value1', 'key2': 'value2'}

    def update(self, data=None, **kwargs):
        self._check_closed()
    
>       self._data.update(data, **kwargs)
E       TypeError: 'NoneType' object is not iterable

..\aiogram\dispatcher\storage.py:411: TypeError
```
## Failure Information (for bugs)

My research showed that for function ```dict.update()``` non-keyword argument MUST be omitted or MUST be iterable. None value is not supported.

### Steps to Reproduce

Test case to be attached to PR

--- a/aiogram/dispatcher/storage.py
+++ b/aiogram/dispatcher/storage.py
@@ -408,6 +408,8 @@ class FSMContextProxy:
     def update(self, data=None, **kwargs):
         self._check_closed()
 
+        if data is None:
+            data = {}
         self._data.update(data, **kwargs)
 
     def pop(self, key, default=None):

--- a/tests/test_dispatcher/test_fsm_context.py
+++ b/tests/test_dispatcher/test_fsm_context.py
@@ -1,0 +1,11 @@
+import pytest
+from aiogram.contrib.fsm_storage.memory import MemoryStorage
+from aiogram.dispatcher import FSMContext
+
+
+class TestFSMContext:
+    @pytest.mark.asyncio
+    async def test_update_data(self):
+        context = FSMContext(MemoryStorage(), chat=1, user=1)
+        async with context.proxy() as data:
+            data.update(key1="value1", key2="value2")

--- a/tests/test_dispatcher/test_fsm_context.py
+++ b/tests/test_dispatcher/test_fsm_context.py
@@ -9,3 +9,6 @@ class TestFSMContext:
         context = FSMContext(MemoryStorage(), chat=1, user=1)
         async with context.proxy() as data:
             data.update(key1="value1", key2="value2")
+        async with context.proxy() as data:
+            assert data['key1'] == "value1"
+            assert data['key2'] == "value2"

